activestorage/package.json

```diff
@@ -16,18 +17,21 @@
   },
   "author": "Javan Makhmali <javan@javan.us>",
   "license": "MIT",
+  "dependencies": {
+    "spark-md5": "^3.0.0"
+  },
   "devDependencies": {
     "babel-core": "^6.25.0",
     "babel-loader": "^7.1.1",
     "babel-preset-env": "^1.6.0",
     "eslint": "^4.3.0",
     "eslint-plugin-import": "^2.7.0",
-    "spark-md5": "^3.0.0",
     "webpack": "^3.4.0"
   },
   "scripts": {
     "prebuild": "yarn lint",
     "build": "webpack -p",
-    "lint": "eslint app/javascript"
+    "lint": "eslint app/javascript",
+    "prepublish": "mkdir -p src && cp app/javascript/activestorage/*.js src/"
```

Comment:
I *think* this should use `prepublishOnly` instead of `prepublish`.
> * prepublish: Run BEFORE the package is packed and published, as well as on local **npm install** without any arguments. 
> * prepublishOnly: Run BEFORE the package is prepared and packed, ONLY on **npm publish**.

https://docs.npmjs.com/misc/scripts

Reply:
Got it - changed to `prepublishOnly` in 266f8b0021d6b7258bceaa4a3434e66746c14638